.circleci/config.yml

```diff
@@ -69,20 +111,35 @@ jobs:
       - run: aws s3 sync frontend/dist s3://qpp-conversion-tool-ui --delete
 
   signal_success:
-    working_directory: ~/qpp-conversion-tool
+    <<: *container_config
     docker:
       - image: circleci/node:7.10.0-browsers
     steps:
-      - checkout
+      - attach_workspace:
+          at: /home/circleci
       - run: sudo apt-get install jq
       - run: ./tools/bin/signal_success.sh
 
 workflows:
   version: 2
   commit:
     jobs:
-      - build
-      - ui_test
+      - checkout
+      - build:
+          requires:
+            - checkout
+      - ui_test:
+          requires:
+            - checkout
+      - deploy_api:
```

Comment:
I'm hesitant to have `deploy_api` only depend on `checkout`.  While the docker image build does do its own independent build of our application, it does not do any tests.  So, this could result in `build` failing, but the `deploy_api` succeeding.  Is that OK?

Reply:
As it stands unit test failures won't be of concern. This job will only run on merges to master.